model/map/src/main/java/org/keycloak/models/map/realm/AbstractRealmEntity.java

```diff
@@ -0,0 +1,1030 @@
+        this.masterAdminClient = masterAdminClient;
+    }
+
+    public String getDefaultRoleId() {
+        return defaultRoleId;
+    }
+
+    public void setDefaultRoleId(String defaultRoleId) {
+        this.updated |= ! Objects.equals(this.defaultRoleId, defaultRoleId);
+        this.defaultRoleId = defaultRoleId;
+    }
+
+    public String getDefaultLocale() {
+        return defaultLocale;
+    }
+
+    public void setDefaultLocale(String defaultLocale) {
+        this.updated |= ! Objects.equals(this.defaultLocale, defaultLocale);
+        this.defaultLocale = defaultLocale;
+    }
+
+    public String getBrowserFlow() {
+        return browserFlow;
+    }
+
+    public void setBrowserFlow(String browserFlow) {
+        this.updated |= ! Objects.equals(this.browserFlow, browserFlow);
+        this.browserFlow = browserFlow;
+    }
+
+    public String getRegistrationFlow() {
+        return registrationFlow;
+    }
+
+    public void setRegistrationFlow(String registrationFlow) {
+        this.updated |= ! Objects.equals(this.registrationFlow, registrationFlow);
+        this.registrationFlow = registrationFlow;
+    }
+
+    public String getDirectGrantFlow() {
+        return directGrantFlow;
+    }
+
+    public void setDirectGrantFlow(String directGrantFlow) {
+        this.updated |= ! Objects.equals(this.directGrantFlow, directGrantFlow);
+        this.directGrantFlow = directGrantFlow;
+    }
+
+    public String getResetCredentialsFlow() {
+        return resetCredentialsFlow;
+    }
+
+    public void setResetCredentialsFlow(String resetCredentialsFlow) {
+        this.updated |= ! Objects.equals(this.resetCredentialsFlow, resetCredentialsFlow);
+        this.resetCredentialsFlow = resetCredentialsFlow;
+    }
+
+    public String getClientAuthenticationFlow() {
+        return clientAuthenticationFlow;
+    }
+
+    public void setClientAuthenticationFlow(String clientAuthenticationFlow) {
+        this.updated |= ! Objects.equals(this.clientAuthenticationFlow, clientAuthenticationFlow);
+        this.clientAuthenticationFlow = clientAuthenticationFlow;
+    }
+
+    public String getDockerAuthenticationFlow() {
+        return dockerAuthenticationFlow;
+    }
+
+    public void setDockerAuthenticationFlow(String dockerAuthenticationFlow) {
+        this.updated |= ! Objects.equals(this.dockerAuthenticationFlow, dockerAuthenticationFlow);
+        this.dockerAuthenticationFlow = dockerAuthenticationFlow;
+    }
+
+    public MapOTPPolicyEntity getOTPPolicy() {
+        return otpPolicy;
+    }
+
+    public void setOTPPolicy(MapOTPPolicyEntity otpPolicy) {
+        this.updated |= ! Objects.equals(this.otpPolicy, otpPolicy);
+        this.otpPolicy = otpPolicy;
+    }
+
+    public MapWebAuthnPolicyEntity getWebAuthnPolicy() {
+        return webAuthnPolicy;
+    }
+
+    public void setWebAuthnPolicy(MapWebAuthnPolicyEntity webAuthnPolicy) {
+        this.updated |= ! Objects.equals(this.webAuthnPolicy, webAuthnPolicy);
```

Comment:
I would prefer it the way it is now as
- It would signal that the object has been updated if either old or new value is `null` and the other non-`null`
- In case both are non-`null` and the name is the same, the `isUpdated()` method would still return the correct value due to updates of `MapWebAuthnPolicyEntity.updated` flag in the `MapWebAuthnPolicyEntity` setters
- setting `updated` to `true` would mean that the realm entity would always be stored in the storage upon commit, even if there has been no change

The same applies to `MapWebAuthnPolicyEntity`.